dotnet/Cucumber.Messages.Specs/MessagesTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Io.Cucumber.Messages.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Cucumber.Messages.Specs
+{
+    public class MessagesTest
+    {
+
+        [Fact]
+        public void ThrowsWhenRequiredFieldsAreMissing()
+        {
+            Assert.Throws<ArgumentNullException>(() => new Background( null, null, null, null, null, null));
```

Comment:
Not sure if you care about linting but space before `(`

Reply:
Corrected.